styles.css

```diff
@@ -0,0 +1,48 @@
+html {
+    font-size: 10px;
+    font-family: Helvetica, sans-serif;
+}
+
+body {
+    background-image: url("game-340574_1280.jpg");
+}
+
+nav {
+    background-color: #195438;
+    padding: 0 20px;
+    display: flex;
+    justify-content: center;
+    font-size: 1.5rem;
+    color: #babfbc;
+}
+
+main{
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    border: #babfbc solid 2px;
+    margin: 30px;
+    border-radius: 20px;
+}
+
+.gameCardsContainer {
+    width: 90%;
```

Comment:
i'm assuming this has 90% width so that it has a bit of space on the sides - it's better to use margin or padding on the left and right and give it a set pixel size, rather than using the dynamic pixel unit of size